src/assets/scripts/client/aircraft/Pilot/Pilot.js

```diff
@@ -701,8 +711,8 @@ export default class Pilot {
             return problematicResponse;
         }
 
-        return [true, `hold ${cardinalDirectionFromFix} of ${fixName}, ` +
-            `${holdParameters.turnDirection} turns, ${holdParameters.legLength} legs`
+        return [true, `hold ${cardinalDirectionFromFix} of ${fixName}, `
```

Comment:
we can just use multi-line strings here:
```js
 return [true, `
     hold ${cardinalDirectionFromFix} of ${fixName},
    ${holdParameters.turnDirection} turns, ${holdParameters.legLength} legs
`];
```

Reply:
Multilining like that actually preserves the line breaks (how strange, right?), so that actually won't work. :disappointed:

Reply:
@n8rzz if you'd like them some other way, just advise, i'm happy to change it :smiley: